common/shlibs

```diff
@@ -830,9 +830,7 @@ libtotem.so.0 libtotem-3.6.2_1
 libclutter-1.0.so.0 clutter-1.14.0_1
 libavfs.so.0 libavfs-1.0.5_1
 libdmr.so.0.1 deepin-movie-reborn-3.2.5_1
-libgjs.so.0 gjs-1.63.3_1
-libgjs-gi.so gjs-1.63.3_1
-libgjs-dbus.so.0 gjs-1.63.3_1
```

Comment:
@sgn  Removed as they are no longer provided by the package.

Reply:
Is there any packages requiring them, right now?

Reply:
Look like no.

Reply:
Yep, nothing depends on them. I think `gjs` stopped providing them quite a while ago.